ej2-react/timepicker/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
 title: Getting started with React Timepicker component | Syncfusion
-description:  Checkout and learn about Getting started with React Timepicker component of Syncfusion Essential JS 2 and more details.
+description: Learn here all about Getting started with React Timepicker component of Syncfusion Essential JS 2 and more details.
```

Comment:
Revert this, remove unwanted space only.

Reply:
Reverted